src/Core/Function.php

```diff
@@ -18,5 +18,10 @@ function base_path(string $path) {
 
 function view(string $path , array $attributes = []){
     extract($attributes);
-    require base_path("Views/Template/{$path}.phtml");
+    require base_path("Views/{$path}.phtml");
+}
+
+function abort($code = 404){
+    require base_path("Views/Template/{$code}.phtml");
```

Comment:
Change the HTTP status code returned to instruct browsers that this is an error response. Also web spiders/crawlers would benefit from this response HTTP status code while indexing for their search engines.

[`http_response_code`](https://www.php.net/manual/en/function.http-response-code.php).

Reply:
Check the HTTP status code for the response using your browser DevTools -> network tab.